gems/smithy/spec/interfaces/client/shapes_spec.rb

```diff
@@ -90,11 +92,9 @@
         end
 
         context 'key and value members' do
-          it 'are shapes of expected member names, shapes and contains traits when applicable' do
-            if shape['key'].nil? && shape['value'].nil?
-              skip("Test does not expect the generated #{id} to have a key/value members")
-            end
+          next if shape['key'].nil? && shape['value'].nil?
 
+          it 'are shapes of expected member names, shapes and contains traits when applicable' do
```

Comment:
not really related to this PR, but this name reads a bit weirdly.  what about `it 'has correct key and value members '` or something like that

Reply:
yeah I definitely think my wording here could be better overall - let me make some adjustments